client/my-sites/plans/jetpack-plans/storage-tier-upgrade/hooks/use-get-storage-upgrade-products.ts

```diff
@@ -1,166 +1,65 @@
+	[ K in PurchasableItemSlug ]: PurchasableItemSlug[];
 };
 
-export const useGetTier2UpgradeProducts = (): StorageUpgradeGetter => {
-	const translate = useTranslate();
-
-	// Security and Backup share the same per-tier storage limits
-	const storageAmount = useJetpack1TbStorageAmountText();
-
-	return useCallback(
-		( slug: string ): SelectorProduct[] => {
-			const features = {
-				items: [
-					{
-						text: translate( '%(storageAmount)s backup storage', { args: { storageAmount } } ),
-						slug: 'jetpack-backup-storage',
-						isHighlighted: true,
-					},
-					{
-						text: translate( 'One-click restore from the past year of backups' ),
-						slug: 'jetpack-one-click-restore',
-						isHighlighted: true,
-					},
-					{
-						text: translate( 'One year activity log*' ),
-						slug: 'jetpack-1-year-archive-activity-log',
-						isHighlighted: true,
-					},
-					{
-						text: translate( 'Real-time backups (as you edit)' ),
-						slug: 'jetpack-real-time-backups',
-					},
-					{
-						text: translate( 'Backups are cloud based and secure' ),
-						slug: 'jetpack-cloud-backups',
-					},
-				],
-			};
-			const product = slugToSelectorProduct( slug );
-
-			return [
-				{
-					...product,
-					displayName: storageAmount,
-					subheader: translate( 'of backup storage' ),
-					buttonLabel: translate( 'Upgrade storage' ),
-					description: translate(
-						'Go back in time and recover all your information for up to a year, with %(storageAmount)s storage space.',
-						{ args: { storageAmount } }
-					),
-					features: features,
-					disclaimer: getJetpackProductDisclaimer(
-						product?.productSlug as ProductSlug | PlanSlug,
-						features.items,
-						getDisclaimerLink()
-					),
-				} as SelectorProduct,
-			];
-		},
-		[ storageAmount, translate ]
-	);
+const UPGRADES: Partial< UpgradesType > = {
+	[ PRODUCT_JETPACK_BACKUP_T1_MONTHLY ]: [
+		PRODUCT_JETPACK_BACKUP_ADDON_STORAGE_10GB_MONTHLY,
+		PRODUCT_JETPACK_BACKUP_ADDON_STORAGE_100GB_MONTHLY,
+		PRODUCT_JETPACK_BACKUP_ADDON_STORAGE_1TB_MONTHLY,
+	],
+	[ PRODUCT_JETPACK_BACKUP_T2_MONTHLY ]: [
+		PRODUCT_JETPACK_BACKUP_ADDON_STORAGE_1TB_MONTHLY,
+		PRODUCT_JETPACK_BACKUP_ADDON_STORAGE_3TB_MONTHLY,
+		PRODUCT_JETPACK_BACKUP_ADDON_STORAGE_5TB_MONTHLY,
+	],
+	[ PRODUCT_JETPACK_BACKUP_ADDON_STORAGE_10GB_MONTHLY ]: [
+		PRODUCT_JETPACK_BACKUP_ADDON_STORAGE_100GB_MONTHLY,
+		PRODUCT_JETPACK_BACKUP_ADDON_STORAGE_1TB_MONTHLY,
+	],
+	[ PRODUCT_JETPACK_BACKUP_ADDON_STORAGE_100GB_MONTHLY ]: [
+		PRODUCT_JETPACK_BACKUP_ADDON_STORAGE_1TB_MONTHLY,
+		PRODUCT_JETPACK_BACKUP_ADDON_STORAGE_3TB_MONTHLY,
+	],
+	[ PRODUCT_JETPACK_BACKUP_ADDON_STORAGE_1TB_MONTHLY ]: [
+		PRODUCT_JETPACK_BACKUP_ADDON_STORAGE_3TB_MONTHLY,
+		PRODUCT_JETPACK_BACKUP_ADDON_STORAGE_5TB_MONTHLY,
+	],
+	[ PRODUCT_JETPACK_BACKUP_ADDON_STORAGE_3TB_MONTHLY ]: [
+		PRODUCT_JETPACK_BACKUP_ADDON_STORAGE_5TB_MONTHLY,
+	],
+	[ PLAN_JETPACK_SECURITY_T1_MONTHLY ]: [ PLAN_JETPACK_SECURITY_T2_MONTHLY ],
```

Comment:
Don't we want to get rid of these two as well?

Reply:
I don't know about that. If you confirm that we do, I'll remove it.

Reply:
Yeah, the pricing page is similarly only showing 10GB for Security, so we shouldn't do the upgrade to 1TB Security here either; they should just buy addon storage.